episodes/severity-static.Rmd

```diff
@@ -0,0 +1,788 @@
+
+[Fraser et al., 2009](https://www.science.org/doi/full/10.1126/science.1176062) reinterpreted the data assessing the biases and getting a clinical severity lower than the 1918 influenza pandemic but comparable with that seen in the 1957 pandemic.
+
+::::::::::::::::::::
+
+:::::::::::::::::::: discussion
+
+Based on your experience:
+
+- Share any previous outbreak in which you participated in its response.
+
+Answer to these questions:
+
+- How did you assess the clinical severity of the outbreak?
+- What were the primary sources of bias?
+- What did you do to take into account the identified bias?
+- What complementary analysis would you do to solve the bias?
+
+:::::::::::::::::::: 
+
+## Delay-adjusted CFR
+
+[Nishiura et al., 2009](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006852) developed a method that considers the **time delay** from the onset of symptoms to death.
+
+Real-time outbreaks may have a number of deaths that are insufficient to determine the time distribution between onset and death. Therefore, we can estimate the _distribution delay_ from historical outbreaks or reuse the ones accessible via R packages like `{epiparameter}` or `{epireview}`, which collects them from literature searches and systematic reviews.
+
+Let's use `{epiparameter}`:
+
+```{r, message=FALSE, warning=FALSE}
+# Get delay distribution
+onset_to_death_ebola <-
+  epiparameter::epidist_db(
+    disease = "Ebola",
+    epi_dist = "onset_to_death",
+    single_epidist = TRUE
+  )
+
+# Plot <epidist> object
+plot(onset_to_death_ebola, day_range = 0:40)
+```
+
+To calculate the delay-adjusted CFR, we can use the `cfr_static()` function with the `data` and `delay_density` arguments.
+
+```{r}
+# Calculate the static CFR while adjusting for delays
+cfr_static(
+  data = ebola1976,
+  delay_density = function(x) density(onset_to_death_ebola, x)
+)
+```
+
+```{r,echo=FALSE}
+out_delay_adjusted <-
+  cfr_static(
+    data = ebola1976,
+    delay_density = function(x) density(onset_to_death_ebola, x)
+  )
+
+out_mean <- out_delay_adjusted %>% pull(severity_mean)
+out_low <- out_delay_adjusted %>% pull(severity_low)
+out_high <- out_delay_adjusted %>% pull(severity_high)
+```
+
+The delay-adjusted CFR indicated that the overall disease severity _at the end of the outbreak_ or with the _latest data available at the moment_ is `r out_mean` with a 95% confidence interval between `r out_low` and `r out_high`, slightly higher than the naive one.
+
+::::::::::::::::::::::::::: spoiler
+
+### How does {cfr} works?
+
+To adjust the CFR, [Nishiura et al., 2009](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006852) use the case and death incidence data to estimate the number of cases with known outcomes:
+
+$$
+  u_t = \dfrac{\sum_{i = 0}^t
+        \sum_{j = 0}^\infty c_{i - j} f_{j}}{\sum_{i = 0} c_i},
+$$
+
+where:
+
+- $c_{t}$ is the daily case incidence at time $t$, 
```

Comment:
In the rendered version the font here is really big compared to the rest of the text

Reply:
is a feature of the workbench. we should fill an issue there to understand the rationale or how to edit locally in our branch